### DISCUSSION TITLE
I was trying to create a Blind app with npx create-nillion-app

### DISCUSSION THREAD

**Discussion by shaikhbazil** [NillionNetwork/discussions#132] [Bugs]
How can i get pass this 
![image](https://github.com/user-attachments/assets/cdd210d3-6266-4238-a36e-a0bdc883b109)

**Comment by crypblizz8** [comment]
Hi there, can you specify which OS you are you using + python version?

As a suggestion, you could run this in a separate terminal
`curl https://nilup.nilogy.xyz/install.sh | bash` and hopefully it fixes it.

**Comment by IkutoJP** [comment]
You need to install according to this guide: https://docs.nillion.com/nillion-sdk-and-tools
